dextra-components/src/components/Marks/Regression/src/Regression.ts

```diff
@@ -0,0 +1,42 @@
+import { customElement, property } from "lit/decorators.js";
+import { PlotMark } from "../../PlotMark/src/PlotMark";
+import * as Plot from "@observablehq/plot";
+import { LinearRegressionOptions } from "@observablehq/plot";
+import { LinearRegressionYOptions } from "@observablehq/plot";
+
+
+@customElement("osl-regression-plot")
+export class OslRegressionPlot extends PlotMark {
+  mark = "regression"
```

Comment:
Repo typescript settings have changed slightly and the repo requires more explicit overrides. For class methods or properties that overwrite the base class, you should use the keyword `override` such as `override mark` - you should see this flagged in the VS Code type hints/errors now.